inspire/modules/deposit/forms.py

```diff
@@ -463,24 +463,28 @@ class LiteratureForm(WebDepositForm):
                                              mapper=journal_title_kb_mapper)
     )
 
-    page_range_article_id = fields.TextField(
+    journal_page = fields.TextField(
         label=_('Page Range/Article ID'),
         description=_('e.g. 1-100'),
+        export_key='page_range_article_id',
         widget_classes="form-control" + ARTICLE_CLASS
```

Comment:
I am wondering about the renaming?

For example, for this field now we would have 3 different names, the form id will be ``journal_page``, then we export that field with the key name ``page_range_article_id`` and then we have to map it to the name ``page_artid`` [here](https://github.com/inspirehep/inspire-next/blob/master/inspire/modules/deposit/workflows/literature.py#L237)

Can we directly use the name in our [inspire.cfg](https://github.com/inspirehep/inspire-next/blob/master/inspire/base/recordext/fields/inspire.cfg) ? Or at least ``export_key`` with that name. And then clean up the ``process_sip_metadata`` function to avoid confusion with so many names for the same thing?